coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/attachment/ProposalDevelopmentNarrativeRule.java

```diff
@@ -84,7 +84,8 @@ public boolean processAddNarrativeBusinessRules(AddNarrativeEvent narrativeEvent
         rulePassed &= getDictionaryValidationService().isBusinessObjectValid(narrative);
         rulePassed &= checkNarrative(document.getDevelopmentProposal().getNarratives(), narrative);
         rulePassed &= validFileNameCharacters(narrative);
-        
+        rulePassed &= validFileType(narrative);
+
         return rulePassed;
     }
```

Comment:
@jwillia Chuck is fixing this doc type issue warning message as part of this jira, can you just fix the STE in your jira we discussed? The STE still needs to be fixed since we should accept doc files.